Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -718,10 +718,6 @@ void BrowserWindow::event(Core::Event& event)
     case GUI::Event::Resize:
         broadcast_window_size(static_cast<GUI::ResizeEvent&>(event).size());
         break;
-    case GUI::Event::KeyDown:
-        // FIXME: add support for hotkeys *within* ladybird
-        event.ignore();
-        break;
```

Comment:
For more details on why we shouldn't `ignore` here, OOPWV explicitly handles this, but only after giving the WebContent a chance to handle the key event:
https://github.com/SerenityOS/serenity/blob/36cd2fb7c5d4099bc9f2ce6d46e02ee221e8dd7c/Userland/Libraries/LibWebView/OutOfProcessWebView.cpp#L378-L388